[API] Add missing inStock field to swagger

arti0090 opened the pull request:
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes (missed thing)
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | Extends #12639
| License         | MIT

I forgot to add this to PR, forgive me sylius 🙏

lchrusciel reviewed (approved):
forgiveness granted

GSadee reviewed (changes requested):
I'm almost sure, that this field has been added to ProductVariant, not Product

arti0090 commented:
Thats true @GSadee i am just changing it

arti0090 commented:
![Zrzut ekranu 2021-05-20 o 09 04 11](https://user-images.githubusercontent.com/35863747/118934174-5b92d500-b94a-11eb-8caf-d96b91396d5c.png)

Changed for ProductVariant, also fixed the missing `price` field

GSadee commented on src/Sylius/Bundle/ApiBundle/Swagger/ProductVariantDocumentationNormalizer.php:
```diff
@@ -26,12 +26,20 @@ public function normalize($object, $format = null, array $context = [])
             'default' => 0,
         ];
 
+        $docs['components']['schemas']['ProductVariant.jsonld-admin.product_variant.read']['properties']['price'] = $priceSchema;
```
`price` field has been also added only for shop endpoints
```suggestion
```

GSadee commented:
Thank you, @arti0090! :tada:
